packages/trpc/server/routers/viewer/teams/removeHostsFromEventTypes.handler.ts

```diff
@@ -28,13 +29,24 @@ export async function removeHostsFromEventTypesHandler({ ctx, input }: RemoveHos
   // check if user has permission to update event types
   if (!hasEventTypeUpdatePermission) throw new TRPCError({ code: "UNAUTHORIZED" });
 
+  // verify that all userIds are members of the team
+  const teamMemberIds = await MembershipRepository.findAcceptedMembershipsByUserIdsInTeam({
+    userIds,
+    teamId,
+  });
+
+  const filteredUserIds = teamMemberIds.map((teamMember) => teamMember.userId);
```

Comment:
This is only called via the UI so members will always belong to the team. If this is called outside of the UI with random userIds this should fail silently.